src/components/radio-button/radio-button-test.stories.tsx

```diff
@@ -131,3 +138,58 @@ export const WithTooltipPositionOnRadioGroup: ComponentStory<
 
 WithTooltipPositionOnRadioGroup.storyName =
   "with tooltip position on RadioGroup";
+
+const radioContainerWidth = 400;
+
+export const RadiobuttonComponent = ({ ...props }) => {
```

Comment:
I think this should be `RadioButtonComponent` in line with the rest of the naming convention.